docs/feature_flags.md

```diff
@@ -218,3 +218,17 @@ Examples of equivalent durations:
 * `(2 ^ 3) * 1m` is the equivalent to `8m` or `480s`
 
 [d2c]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/deltatocumulativeprocessor
+
+## Use Uncached IO
+
+`--enable-feature=use-uncached-io`
+
+Only available on Linux.
+
+When enabled, it forces qualifying I/O operations to bypass the page cache. Its primary
```

Comment:
Can we find a better way to explain this?  
I suspect most people reading this will not know what "qualifying I/O operations" are.

Would it help to add the word "experimental" ?

Reply:
pushed a new version